goes.go

```diff
@@ -342,6 +342,27 @@ func (c *Connection) Delete(d Document, extraArgs url.Values) (*Response, error)
 	return r.Run()
 }
 
+// Enable use number behaviour
+func (c *Connection) UseNumberEnable() {
```

Comment:
This method should just be called `UseNumber` to reflect the underlying method which will be called on the json decoder. This means that the attribute will need to be renamed to not conflict, but it should probably be unexported anyway (so `useNumber`, probably).